ciao-4.11/contrib/bin/ecf_calc

```diff
@@ -365,10 +364,15 @@ def get_ecf(infile,reg,outfile,x,y,radius,binsize,fraction,clobber):
     ecf.close()
 
 
-@hide_chips_commands()
 def plot_ecf(rad_file):
 
-    import pychips
+    import matplotlib
+    from matplotlib import pyplot as plt
+    from matplotlib.ticker import ScalarFormatter
+
+    matplotlib.use("TkAgg")
+    matplotlib.interactive(True)
+    matplotlib.rcParams["toolbar"] = "None"    
```

Comment:
I'm too lazy to RTFD, but is this script intended to just create an output file, or display the data for the user to see before exiting the script.

Reply:
It's just a plot to the screen, there's no output to file.

Reply:
Is there a reason to turn the toolbar off? Users might want to be able to save a png or zoom around in the figure (or am I over-thinking it)?